### DISCUSSION TITLE
Slats not in T.O config...

### DISCUSSION THREAD

**Discussion by Adi-18** [flybywiresim/aircraft#4846] [Help]
... after update of FBW A32. What dose it mean an how to fix it?
![image](https://user-images.githubusercontent.com/29123730/118503381-dd0c1c80-b72a-11eb-86c9-f3808c88e2c3.png)

**Comment by MapleNAV-TCASFAULT** [comment]
It looks like you accidentally set takeoff thrust. Simply retract your flaps to 2 or 1+F. It also looks like your spoilers are open so simply close them. If you still need to be in the configuration that you're in right now there is a CLR button near ecam page buttons, just press it and it'll dismiss those warnings.
